styleguide/_themes/derek/scss/components/subnav.scss

```diff
@@ -1,91 +1,144 @@
-//subnav
 .subnav {
-  background-color: $blue-gray;
+  background-color: $gray-dark;
   border-radius: 0;
+}
 
-  .nav > li > a:focus {
-    background-color: transparent;
-  }
+.subnav-container {
+  @include make-container;
+}
 
-  .nav > li.no-hover > a:focus {
-    background-color: $brand-success;
-  }
+.subnav-links {
+  list-style-type: none;
+  padding: 0;
+}
 
-  .subnav-inside {
-    width: 100%;
-  }
+.subnav-header {
+  float: left;
+  width: $full-width;
+}
 
-  .navbar-header {
+.subnav-collapseBtn {
+  @include make-xs-column(1);
+}
 
-    .col-xs-10 {
-      margin-top: 10px;
-    }
+.subnav-toggleBtn {
+  margin: 14px 0;
+  padding: 0;
+}
 
-    button {
-      background-color: transparent;
-    }
+.subnav-pageTitleContainer {
+  @include make-xs-column(11);
+  display: none;
+  margin-top: 5px;
+}
 
-  }
+.subnav-pageTitle {
+  color: $white;
+}
 
-  .button {
-    margin: 7px 0;
-  }
+.icon-bar {
+  background-color: $white;
+}
+
+.subnav-collapseContainer {
+  padding: 0 1px;
+}
 
-  .icon-bar {
-    background-color: $white;
+.subnav-item {
+  border-bottom: solid transparent 4px;
+  float: left;
+  height: 50px;
+  padding: 10px 18px;
+
+  &:hover {
+    background-color: transparent;
+    border-bottom: solid $teal-dark 4px;
   }
 
-  li {
-    border-bottom: solid transparent 4px;
-    float: left;
-    padding: 7px 7px 6px;
+  &:focus {
+    background-color: transparent;
+  }
+}
 
-    &:hover {
-      background-color: transparent;
-      border-bottom: solid $teal-dark 4px;
-    }
+.subnav-link {
+  color: $white;
+  font-weight: 300;
+  text-decoration: none;
 
-    a {
-      color: $white;
-      font-weight: 400;
-      text-decoration: none;
+  &:hover {
+    background-color: transparent;
+    color: $white;
+    text-decoration: none;
+  }
+}
 
-      &:hover {
-        background-color: transparent;
-        text-decoration: none;
-      }
-    }
+.subnav-selectedLink {
+  background-color: transparent;
+  border-bottom: solid $teal-dark 4px;
+}
 
-    &.no-hover {
-      float: right;
+.subnav-leadBtn {
+  @include button;
+  @include button-lead;
+  padding: 8px 10px;
+}
 
-      &:hover {
-        border-bottom: solid transparent 4px;
-      }
+.subnav-noHover > a {
```

Comment:
@WidgetsBurritos Sorry, I thought you were refering to this entire selector, not just the focus. The focus was added because we encountered an issue with the button in www and had to add it when we originally built the subnav. I will fix the color and text decoration though.